tests/load-tests/cluster_read_config.yaml

```diff
@@ -99,6 +99,26 @@
   env_variable: {{ var }}
 {% endfor %}
 
+# Git info
+{% macro git_info(path, dir) -%}
+- name: metadata.git.{{ path }}.commit.hash
+  command: cd "{{ dir }}" && git log -1 --pretty=format:"%H"
+- name: metadata.git.{{ path }}.commit.abbreviated_hash
+  command: cd "{{ dir }}" && git log -1 --pretty=format:"%h"
+- name: metadata.git.{{ path }}.command.author_date
+  command: cd "{{ dir }}" && git log -1 --pretty=format:"%aI"
+- name: metadata.git.{{ path }}.command.committer_date
+  command: cd "{{ dir }}" && git log -1 --pretty=format:"%cI"
+- name: metadata.git.{{ path }}.command.subject
+  command: cd "{{ dir }}" && git log -1 --pretty=format:"%s"
+- name: metadata.git.{{ path }}.command.author_name
+  command: cd "{{ dir }}" && git log -1 --pretty=format:"%aN"
+- name: metadata.git.{{ path }}.command.author_email
+  command: cd "{{ dir }}" && git log -1 --pretty=format:"%aE"
```

Comment:
```suggestion
- name: metadata.git."{{ path }}".commit.hash
  command: cd "{{ dir }}" && git log -1 --pretty=format:"%H"
- name: metadata.git."{{ path }}".commit.abbreviated_hash
  command: cd "{{ dir }}" && git log -1 --pretty=format:"%h"
- name: metadata.git."{{ path }}".command.author_date
  command: cd "{{ dir }}" && git log -1 --pretty=format:"%aI"
- name: metadata.git."{{ path }}".command.committer_date
  command: cd "{{ dir }}" && git log -1 --pretty=format:"%cI"
- name: metadata.git."{{ path }}".command.subject
  command: cd "{{ dir }}" && git log -1 --pretty=format:"%s"
- name: metadata.git."{{ path }}".command.author_name
  command: cd "{{ dir }}" && git log -1 --pretty=format:"%aN"
- name: metadata.git."{{ path }}".command.author_email
  command: cd "{{ dir }}" && git log -1 --pretty=format:"%aE"
```
let's surround the path with quotes to avoid getting the JSON tree crazy such as [this](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/redhat-appstudio_e2e-tests/1085/pull-ci-redhat-appstudio-e2e-tests-main-load-test-ci-10u-10t/1770013564371734528/artifacts/load-test-ci-10u-10t/redhat-appstudio-load-test/artifacts/load-tests.json) (for the case of the `.` path which translates to `.git...command.has`):
```json
"git": {
  "": {
   "": {
    "command": {
      "author_date": null,
      "author_email": null,
      "author_name": null,
      "committer_date": null,
      "subject": null
    },
    "commit": {
      "abbreviated_hash": null,
      "hash": null
    }
  }
},
"tmp/infra-deployments": {
  "command": {
    "author_date": null,
    "author_email": null,
    "author_name": null,
    "committer_date": null,
    "subject": null
  },
  "commit": {
    "abbreviated_hash": null,
    "hash": null
  }
}
},
```

with my suggestion the actual path in JSON would be `.git.".".command.has` which is what we want.

Reply:
got it! ack